lib/chef/event_dispatch/dispatcher.rb

```diff
@@ -20,6 +20,14 @@ def register(subscriber)
         @subscribers << subscriber
       end
 
+      # Check to see if we are dispatching to a formatter
+      def formatter?
+        @subscribers.each do |s|
```

Comment:
Agreed, I think `any?` is the correct logic here. On my first reading I mistakenly thought this was logically equivalent to that (after some coffee I see that it's not).